test/features/mailbox/presentation/extensions/list_mailbox_node_extension_test.dart

```diff
@@ -94,4 +94,187 @@ void main() {
       expect(ids, ['1']);
     });
   });
+
+  group('ListMailboxNodeExtension::insertAfterStarredOrInbox', () {
```

Comment:
we should add test case to add virtual box duplicated?

Reply:
It already has duplicate test cases.